src/sentry/sentry_metrics/kafka_metrics_interface.py

```diff
@@ -0,0 +1,42 @@
+from typing import Mapping
+
+from sentry.sentry_metrics.metrics_interface import GenericMetricsBackend
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
+
+
+class KafkaGenericMetricsBackend(GenericMetricsBackend):
+    def __init__(self) -> None:
+        super().__init__()
+
+    def counter(
+        self,
+        use_case_id: UseCaseID,
+        org_id: int,
+        project_id: int,
+        metric_name: str,
+        value: int,
```

Comment:
Curious if this should support float. Floats can be very useful for things like timings, which don't look like they are easily supported otherwise.

Reply:
You're referring to sets and distributions supporting float, right?